runtime/lua/vim/lsp.lua

```diff
@@ -1567,7 +1571,23 @@ function lsp.start_client(config)
     if method ~= 'textDocument/didChange' then
       changetracking.flush(client)
     end
-    return rpc.notify(method, params)
+
+    local result = rpc.notify(method, params)
+
+    if result then
```

Comment:
If result is fasly (i.e. rpc.notify failed?) is that reflected in the logs somehow? If not, should it be?

Reply:
The result is false if the client stopped.
It doesn't indicate that the notification failed.

Reply:
I applied that suggestion in the PR here: https://github.com/neovim/neovim/pull/24411